### DISCUSSION TITLE
scrollIntoView aborting when using in combination with x-show

### DISCUSSION THREAD

**Discussion by simonferndriger** [alpinejs/alpine#4321] [7. Bugs (archived)]
When you want to scroll to an element that you just made visible with `x-show`, it doesn't work with `scrollIntoView()` even when using `$nextTick()`. It starts scrolling for a split second and then stops. However, in the very same scenario, when the element is alreayd visible (or if you call `scrollIntoView()` a second time), it works fine.

**Comment by ekwoka** [comment]
Can you demo this? https://awesomealpine.com/play

**Comment by simonferndriger** [comment]
Sure, here we go: https://codepen.io/simonferndriger/pen/oNrBbam

**Reply by simonferndriger** [reply]
Sorry, on CodePen it works, when using `$nextTick`. Guess I can currently only reproduce it in my own app. Probably a multi-factor issue? However, I just use an empty `<a x-ref="...">` element now that is always visible, so I'm fine with that.

**Reply by ekwoka** [reply]
I'd imagine there is something else running that is changing the scroll target so it gives up.

**Comment by ludwig-gramberg** [comment]
Same issue as: https://github.com/alpinejs/alpine/discussions/4580
